### PR TITLE
Update config-mattermost-server.rst

### PR DESCRIPTION
**Pull request by Sayanta66**
Did minor changes

### CODE REVIEW

File: source/install/config-mattermost-server.rst [hunk 1]
```diff
@@ -40,8 +40,8 @@ Create the System Admin user and set up Mattermost for general use.
  
 .. note::
 
-    * Files and images that users attach to their messages are not stored in the database. Instead, they're stored in a location that you specify. You can store the files on the local file system or in Amazon S3.
-    * Make sure that the location has enough free space. The amount of storage that's required depends on the number of users and on the number and size of files that users attach to messages.
+    * Files and images that users attach to their messages are not stored in the database. Instead, they're stored in a location that you specify. You can keep the files on the local file system or in Amazon S3.
```
Comment (justinegeffen) [inline source/install/config-mattermost-server.rst:43]:
```suggestion
    * Files and images that users attach to their messages are not stored in the database. Instead, they're stored in a location that you specify. You can store the files on the local file system or in Amazon S3.
```

Comment (justinegeffen) [reply source/install/config-mattermost-server.rst:43]: Your suggestion made me rethink this sentence, thank you!

**Review by justinegeffen — COMMENTED** [review]
A very helpful PR, thank you! I've added a suggestion - let me know what you thoughts are and if you're open to committing the change. :)

File: source/install/config-mattermost-server.rst (hunk 1 above)
Comment (Sayanta66) [reply source/install/config-mattermost-server.rst:43]: Thanks a lot

File: source/install/config-mattermost-server.rst [hunk 2]
```diff
@@ -40,8 +40,8 @@ Create the System Admin user and set up Mattermost for general use.
  
 .. note::
 
-    * Files and images that users attach to their messages are not stored in the database. Instead, they're stored in a location that you specify. You can store the files on the local file system or in Amazon S3.
-    * Make sure that the location has enough free space. The amount of storage that's required depends on the number of users and on the number and size of files that users attach to messages.
+    * Files and images that users attach to their messages are not stored in the database. Instead, they're stored in a location that you specify such as the local file system or in Amazon S3.
```
Comment (cwarnermm) [inline source/install/config-mattermost-server.rst:43]:
```suggestion
    * Files and images that users attach to their messages are not stored in the database. Instead, they're stored in a location that you specify, such as the local file system or in Amazon S3.
```

**Review by cwarnermm — COMMENTED** [review]
Thank you for this contribution! I've provided a punctuation suggestion inline.

**Review by cwarnermm — APPROVED** [review]
Thank you for this docs contribution!

**Pull request merged**